ecs-agent/netlib/platform/common_linux.go

```diff
@@ -63,8 +63,14 @@ const (
 	// It is assigned 100 because it is an unrealistically high
 	// value for interface index.
 	indexHighValue = 100
+
+	// srcBRResolv is the path of resolv.conf file on a Bottlerocket host
+	// that we copy to the task namespace when using debug mode.
+	srcBRResolv = "/run/netdog/resolv.conf"
 )
 
+var srcResolv = "/etc/resolv.conf"
```

Comment:
Instead of making this a var, I would suggest keeping this a const. You can use a local var in the function instead of creating a global var if needed.

Reply:
Ack, will make that change